modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -527,3 +529,29 @@ function editConsentStatusFields($db, $user)
         }
     }
 }
+
+/**
+ * Handles the updating of candidate's date of birth.
+ *
+ * @param Database $db   database object
+ * @param User     $user user object
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editCandidateDOB(\Database $db, \User $user): void
+{
+    if (!$user->hasAnyPermission('candidate_parameter_edit', 'candidate_dob_edit')) {
```

Comment:
```suggestion
    if (!$user->hasAnyPermission(array('candidate_parameter_edit', 'candidate_dob_edit'))) {
```
This function actually takes an array, not strings, as an argument. This should fix your Travis/phan error.

Reply:
Okay thanks!